frontend/public/components/masthead-toolbar.jsx

```diff
@@ -318,13 +319,27 @@ class MastheadToolbarContents_ extends React.Component {
           component: <Link to="/quickstart">{t('public~Quick Starts')}</Link>,
         },
         {
-          label: t('public~Documentation'),
+          label: multicluster ? t('public~OCP Documentation') : t('public~Documentation'),
```

Comment:
We'll need to adjust the label here based on the branding (OKD/OCP/OSD/etc).

Reply:
Wondering if it makes sense to leave this simply labelled `Documentation` and differentiate the addition of ACM with the planned `ACM Documentation` so we don't have to do such complex switching with the product name?  What do you think @andybraren, @bmignano?

Reply:
Good catch. `Documentation` and `ACM Documentation` works, although I wonder if `OpenShift Documentation` would be even clearer while still being generic enough to cover all three.

By the way, does this link point to the [OKD Docs](https://docs.okd.io/latest/welcome/index.html) when the Console is running on OKD, or does it always point to OpenShift's docs today?

Reply:
> By the way, does this link point to the [OKD Docs](https://docs.okd.io/latest/welcome/index.html) when the Console is running on OKD, or does it always point to OpenShift's docs today?

For OKD, the link points to `https://docs.okd.io/latest/`, so we'd still have to differentiate between OpenShift and OKD.  Leaving the label as `Documentation` for all eliminates the need to do *any* label switching.

Reply:
Works for me.